v3/src/components/graph/adornments/univariate-measures/univariate-measure-adornment-model.ts

```diff
@@ -5,11 +5,9 @@ import { IDataConfigurationModel } from "../../../data-display/models/data-confi
 import { ICase } from "../../../../models/data/data-set-types"
 
 export const MeasureInstance = types.model("MeasureInstance", {
-  labelCoords: types.maybe(PointModel)
+  labelCoords: types.maybe(PointModel),
+  value: types.optional(types.number, 0)
```

Comment:
We've been here before, but do we need to store the value persistently? I would have thought that it could be computed when needed but apparently there's something here I don't understand.

Reply:
@kswenson With the previous configuration where value is volatile and set to 0 by default, if you add the Mean or Median adornment, then undo adding it, then redo adding it, the adornment returns with a value of 0.  It seems when the action is redone, `adornmentStore`'s`addAdornment` action isn't triggered, which means the adornment model's `updateCategories` isn't being called. And so the instance of the adornment doesn't get a real value set.

There's probably a way to have the volatile value properly set when the adornment returns, but it's not obvious to me. I'll see if I can work something out, though.